person_capture/updater.py

```
@@ -679,7 +713,11 @@ def _run():
                     return
                 self.info.emit("Checking for updates…")
                 # path 1: git
-                if is_git_repo(self.repo):
+                if is_git_checkout(self.repo):
+                    if not is_git_repo(self.repo):
+                        s.setValue("update_last_check_t", _now()); s.sync()
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)